adafruit_aw9523.py

```diff
@@ -1,4 +1,5 @@
 # SPDX-FileCopyrightText: Copyright (c) 2021 ladyada for Adafruit
+# SPDX-FileCopyrightText: Copyright (c) 2021 ladyada for Adafruit
```

Comment:
Did this line get duplicated accidentally? I think we don't need two copies of this copyright.

Reply:
oops - ctrl - D in pycharm duplicates the current line I think - it sometimes gets accidently pressed but I must have missed this one